linux-tkg-config/6.5/minimal-modprobed.db

```diff
@@ -109,28 +109,29 @@ cuse
 dca
 des_generic
 dm_crypt
-dmi_sysfs
 dm_log
 dm_mirror
 dm_mod
 dm_multipath
 dm_region_hash
 dm_round_robin
+dmi_sysfs
 dns_resolver
 drm
 drm_buddy
 drm_display_helper
+drm_dp_helper
```

Comment:
If I remember well, this module is not recognized when actually building the kernel, try to build the kernel first and look for the log messages it outputs about modules that are not recognized. These don't need to be there.

Reply:
Sorry for taking so long to respond. Indeed, I do get a warning saying `drm_dp_helper config not found!!`. I'll remove that one then.

Reply:
Hmm, I've just noticed that it does also build the module though... `CC [M]  drivers/gpu/drm/display/drm_dp_helper.o`